nodejs/src/index.ts

```diff
@@ -461,3 +459,22 @@ export class SpeechifyAccessTokenManager {
 		);
 	}
 }
+
+const base64ToBlob = async (data: string, audioFormat: AudioSpeechFormat) => {
+	const mime = audioFormatToMime(audioFormat);
+
+	if (typeof Buffer !== "undefined") {
+		return new Blob([Buffer.from(data, "base64")], { type: mime });
+	}
+
+	// Browser trick
+	const base64Response = await fetch(`data:${mime};base64,${data}`);
```

Comment:
what is advantage of doing this compating to ``data:${mime};base64,${btoa(data)}``?

Reply:
`btoa` returns a string of bytes, so it needs further conversion to a blob or Uint8Array, and to do it cross-browser you do a loop and read char codes from this "string".

This trick delivers the blob directly. I did quick research before implementing it, seems to be the easiest way, supported everywhere